common/src/main/java/io/druid/metadata/DefaultPasswordProvider.java

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
Would you mind updating the license header to follow the template here? https://github.com/druid-io/druid/blob/master/processing/src/main/java/io/druid/segment/NullDimensionSelector.java#L1

This is in line with what the ASF does for their own projects http://www.apache.org/legal/src-headers.html#headers